pySDC/implementations/convergence_controller_classes/estimate_embedded_error.py

```diff
@@ -1,6 +1,6 @@
 import numpy as np
 
-from pySDC.core.ConvergenceController import ConvergenceController
+from pySDC.core.ConvergenceController import ConvergenceController, _Pars
```

Comment:
If you have to import an underscore method, better not call it an internal one (which is what the underscore means).